packages/blockchain-wallet-v4/src/redux/payment/nfts/utils.ts

```diff
@@ -1072,7 +1076,7 @@ export async function _makeSellOrder({
     basePrice: new BigNumber(basePrice.toString()),
     calldata,
     englishAuctionReservePrice: reservePrice ? new BigNumber(reservePrice.toString()) : undefined,
-    exchange: '0x7Be8076f4EA4A4AD08075C2508e481d6C946D12b',
+    exchange: '0x7Be8076f4EA4A4AD08075C2508e481d6C946D12b'.toLowerCase(),
```

Comment:
Can you add a comment here since this isn't usual practice

Reply:
I will update it to be hardcoded to lower case instead, my bad for the hacky fix!

Reply:
All good!